docs/docs/train-tutorial.md

```diff
@@ -0,0 +1,338 @@
+# User Tutorial: Training a Model on Labaled Videos
+
+This section walks through how to train a model using `zamba`. If you are new to `zamba` and just want to classify some videos as soon as possible, see the [Quickstart](quickstart.md) guide.
+
+This tutorial goes over the steps for using `zamba` if:
+
+* You already have `zamba` installed (for details see the [Installation](install.md) page)
+* You have labeled videos that you want to use to train or finetune a model
+
+`zamba` can run two types of model training:
+
+* Fine-tuning a model with labels that are a subset of the possible [zamba labels](models.md#species-classes)
+* Retraining a model to predict an entirely new set of labels
+
+The process is the same for both cases.
+
+## Basic usage: command line interface
+
+Say that we want to finetune the `time_distributed` model based on the videos in `example_vids` and the labels in `example_labels.csv`. 
+
+Minimum example for training in the command line:
+
+```console
+$ zamba train --data-dir example_vids/ --labels example_labels.csv
+```
+
+### Required arguments
+
+To run `zamba train` in the command line, you must specify both `--data-directory` and `--labels`.
+
+* **`--data-dir PATH`:** Path to the folder containing your labeled videos.
+* **`--labels PATH`:** Path to a CSV containing the video labels to use as ground truth during training. There must be columns for both filepath and label. 
```

Comment:
same as above - let's mention the optional columns since the split (train/val/holdhout) column is important (if they don't include a split col, they can optionally use the `split_proportions` argument)